### PR TITLE
Added extra info to readme building steps

### PR DESCRIPTION
**Pull request by iamgabrielma**
### Fix

This PR just adds extra information to the Simplenote iOS building steps `README.md` file, specially around the credentials setup and how to resolve the inability to build a project due to missing packages. These clarifications came out from the blocks I faced when trying to build the app as an OSS contributor and may help others.

These changes do not require release notes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,19 +53,21 @@ You can also open the project by double clicking on Simplenote.xcworkspace file,
...
 ```
 
-Then edit the new `Simplenote/Credentials/SPCredentials.swift` file and change the `simperiumAppID` and `simperiumApiKey` fields to the correct values for your new app.
+This will copy the demo SPCredentials file into the correct directory with the basic details for an OSS contributor. Then edit the new `Simplenote/Credentials/SPCredentials.swift` file and change the `simperiumAppID` and `simperiumApiKey` fields to the correct values that appear in your Simperium app. 
 
-This will allow you to compile and run the app on a device or a simulator.
+If the build fails with an "Account Creation Failed" error and Xcode asks for your GitHub credentials most likely you need to update the Swift Packages before being able to build the app. Package issues need to be resolved before you can build the app, so the credentials file is put in place. For this you can setup GitHub under Xcode > Preferences > Accounts, then enter your account and access token. If you have 2FA on your GitHub account you won't be able to connect to without the SSH key.
```
Comment (charliescheer) [inline README.md]: I think it might be helpful here to add a link to the GH instructions for how to get the access token and how to setup the ssh key

Comment (charliescheer) [reply README.md]:
```suggestion
If the build fails with an "Account Creation Failed" error and Xcode asks for your GitHub credentials most likely you need to update the Swift Packages before being able to build the app. Package issues need to be resolved before you can build the app, so the credentials file is put in place. For this you can setup GitHub under Xcode > Preferences > Accounts, then enter your account and [access token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token). If you have 2FA on your GitHub account you won't be able to connect to without the [SSH key.](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
```

**Review by charliescheer — COMMENTED** [review]
Thanks for adding these additional details to the readme!

File: README.md (hunk 1 above)
Comment (iamgabrielma) [reply README.md]: Those are great additions 👍 , thanks!

**Review by charliescheer — APPROVED** [review]
Looks good to me.  Thanks for making these changes!

**Pull request merged**